presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -316,50 +316,50 @@ the ``org.apache.hadoop.conf.Configurable`` interface from the Hadoop Java API,
 will be passed in after the object instance is created and before it is asked to provision or retrieve any
 encryption keys.
 
-Querying Hive Tables
---------------------
+Examples
+--------
 
-The following table is an example Hive table from the `Hive Tutorial`_.
-It can be created in Hive (not in Presto) using the following
-Hive ``CREATE TABLE`` command:
+The Hive connector supports querying and manipulating Hive tables and schemas
+(databases). While some uncommon operations will need to be performed using
+Hive directly, most operations can be performed using Presto.
 
-.. _Hive Tutorial: https://cwiki.apache.org/confluence/display/Hive/Tutorial#Tutorial-UsageandExamples
+Create a new Hive schema named ``web`` that will store tables in an
+S3 bucket named ``my-bucket``::
 
-.. code-block:: none
+    CREATE SCHEMA hive.web
+    WITH (location = 's3://my-bucket/')
 
-    hive> CREATE TABLE page_view (
-        >   viewTime INT,
-        >   userid BIGINT,
-        >   page_url STRING,
-        >   referrer_url STRING,
-        >   ip STRING COMMENT 'IP Address of the User')
-        > COMMENT 'This is the page view table'
-        > PARTITIONED BY (dt STRING, country STRING)
-        > STORED AS SEQUENCEFILE;
-    OK
-    Time taken: 3.644 seconds
+Create a new Hive table named ``page_views`` in the ``web`` schema
+that is stored using the ORC file format, partitioned by date and
+country, and bucketed by user into ``50`` buckets::
 
-Assuming that this table was created in the ``web`` schema in
-Hive, this table can be described in Presto::
+    CREATE TABLE hive.web.page_views (
+      view_time timestamp,
+      user_id bigint,
+      page_url varchar,
+      ds date,
+      country varchar
+    )
+    WITH (
+      format = 'ORC',
+      partitioned_by = ARRAY['ds', 'country'],
```

Comment:
Can you add a note that the partitioning column(s) need to be last in the table definition?